### PR TITLE
Backend auth models, better manage.py shell

### PR DESCRIPTION
**Pull request by disconnect3d**
List of changes can be seen in the commit logs.

Please, review particular commits as I tried to keep them clean/seperate.

### CODE REVIEW

**Comment by disconnect3d** [comment]
Screen of fancy new `manage.py shell`:
fancy_shell

I am not sure what to do with tests launching, currently, for me it requires to change `PYTHONPATH`; otherwise it doesn't find the proper paths...

Before changing `PYTHONPATH`:

``` bash
(flask) dc@dc:/media/sf_D_DRIVE/projects/TermsSchedulerFlask/backend/src$ echo $PYTHONPATH
/home/dc/lib/python-2.7/site-packages
(flask) dc@dc:/media/sf_D_DRIVE/projects/TermsSchedulerFlask/backend/src$ py.test
Traceback (most recent call last):
  File "/home/dc/.virtualenvs/flask/local/lib/python2.7/site-packages/_pytest/config.py", line 320, in _importconftest
    mod = conftestpath.pyimport()
  File "/home/dc/.virtualenvs/flask/local/lib/python2.7/site-packages/py/_path/local.py", line 650, in pyimport
    __import__(modname)
  File "/media/sf_D_DRIVE/projects/TermsSchedulerFlask/backend/src/tests/conftest.py", line 5, in <module>
    from application import create_app
ImportError: No module named application
ERROR: could not load /media/sf_D_DRIVE/projects/TermsSchedulerFlask/backend/src/tests/conftest.py
```

After changing `PYTHONPATH`:

``` bash
(flask) dc@dc:/media/sf_D_DRIVE/projects/TermsSchedulerFlask/backend/src$ PYTHONPATH=.
(flask) dc@dc:/media/sf_D_DRIVE/projects/TermsSchedulerFlask/backend/src$ py.test
==================================================================================== test session starts =====================================================================================
platform linux2 -- Python 2.7.10, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /media/sf_D_DRIVE/projects/TermsSchedulerFlask/backend/src, inifile: 
plugins: flask-0.10.0
collected 7 items 

tests/test_authorization.py .......

================================================================================== 7 passed in 0.66 seconds ==================================================================================
```

File: backend/src/application/authorization/models.py [hunk 1]
```diff
@@ -39,6 +46,21 @@ def verify_auth_token(token):
             return None  # invalid token
         return User.query.get(data['id'])
 
+    def __repr__(self):
+        return "{id}: '{first_name}' '{last_name}', '{email}', admin: {is_admin}".format(**self.__dict__)
```
Comment (Alexander3) [inline backend/src/application/authorization/models.py:50]: `.format(**self.__dict__)` - nice

File: backend/src/manage.py [hunk 2]
```diff
@@ -19,13 +19,63 @@
 manager.add_command('db', MigrateCommand)
 manager.add_command('create_admin', CreateAdmin())
 
+BLUE = '\033[94m'
```
Comment (Alexander3) [inline backend/src/manage.py]: Maybe use https://pypi.python.org/pypi/termcolor

**Comment by disconnect3d** [comment]
As suggested in https://github.com/disconnect3d/TermsScheduler/pull/3#discussion_r70981128 changed the `Enchance manage.py shell so that it imports auth models` to use `termcolors`.

The proposed solution is prettier and works on both Windows and Linux.

Linux screen:
termcolor_lin

Windows screen:
termcolor_win

That's cool :+1:.

**Pull request merged**